<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1234</width>
    <height>994</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task6</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="Stepper_Motor_IR">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>1211</width>
      <height>911</height>
     </rect>
    </property>
    <property name="title">
     <string>Servo/Potentiometer</string>
    </property>
    <widget class="QRadioButton" name="Choose_IR">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>141</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>IR Reading</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_Stepper_Position">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>330</y>
       <width>241</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stepper Position</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_Stepper_Velocity">
     <property name="geometry">
      <rect>
       <x>820</x>
       <y>330</y>
       <width>221</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Stepper Velocity (0-100)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Stepper_Velocity_In">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>330</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="IR_Out" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="Stepper_Position_Out" native="true">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>360</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="Stepper_Velocity_Out" native="true">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>360</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="Stepper_Position_In">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>330</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_Encoder">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>630</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Encoder Reading</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QWidget" name="Encoder_Out" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>660</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_DC_Position">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>630</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Position (0-360)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DC_Position_In">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>630</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="DC_Position_Out" native="true">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>660</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_DC_Velocity">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>630</y>
       <width>191</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Velocity (0-400)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="DC_Velocity_In">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>1070</x>
       <y>630</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="DC_Velocity_Out" native="true">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>660</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="Pot_Out" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="Servo_Position_Out" native="true">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>60</y>
       <width>391</width>
       <height>231</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_Pot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>231</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Potentiometer Reading</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="Choose_Servo_Position">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>211</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Servo Position (0-179)</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="Servo_Position_In">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>30</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>Stepper Motor/IR</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>600</y>
       <width>161</width>
       <height>19</height>
      </rect>
     </property>
     <property name="text">
      <string>DC Motor/Encoder</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>685</x>
       <y>310</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>(-360-360)</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="actionConnect">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="actionConfigure">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>950</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Configure</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
